jsonnet/components/alertmanager.libsonnet

```diff
@@ -156,6 +154,30 @@ function(params)
       }],
     },
 
+    kubeRbacProxyWebSecret: {
+      apiVersion: 'v1',
+      kind: 'Secret',
+      metadata: {
+        name: 'alertmanager-kube-rbac-proxy-web',
+        namespace: cfg.namespace,
+        labels: { 'app.kubernetes.io/name': 'alertmanager-main' },
+      },
+      type: 'Opaque',
+      stringData: {
+        'config.yaml': std.manifestYamlDoc({
+          authorization: {
+            resourceAttributes:
+              {
+                apiGroup: 'monitoring.coreos.com',
+                resource: 'alertmanagers',
```

Comment:
to be consistent with the tenant-aware [kube-rbac-proxy config](https://github.com/openshift/cluster-monitoring-operator/blob/38a487897d146715c734d11e80a2b60e7c374eb0/assets/alertmanager/kube-rbac-proxy-secret.yaml#L10-L17), it may be better to authorize based on the permissions to the `prometheusrules` resource.

Reply:
ok, the permission to manipulate prometheusrules is more logic than alertmanagers. I think the kube-rbac-proxy before thanos can  use this new RBAC settings, too.